web/save-svg-as-png.d.ts

```diff
@@ -0,0 +1,14 @@
+/** Declaration file generated by dts-gen */

```

Comment:
Is is possible to put this file in a subfolder instead of the root web folder?

Reply:
As far as I see we need an `index.d.ts` file in root `web` folder that imports others definitions files from anywhere and then export them.
For now we only have this one.

Would it be ok for you if I add this `index.d.ts` file in root and put `save-svg-as-png.d.ts` in a new `types` folder ?

Reply:
yeah, it's Ok. Not a big problem from my side if you think it's better to keep the `save-sgv-as-png` file in the `web` folder, if you think its a common/good practice for this kind of projects.

Reply:
at the end there is already a `typedefs` folder for `protocol-numbers` @mariomac so I moved `save-sgv-as-png` to that folder